bids/variables/tests/test_variables.py

```diff
@@ -202,3 +202,37 @@ def test_resampling_edge_case(tmpdir, TR, nvols):
     dense_var = coll.variables['val'].to_dense(coll.sampling_rate)
     regressor = dense_var.resample(1.0 / TR).values
     assert regressor.shape == (nvols, 1)
+
+
+@pytest.mark.parametrize(
+    "TR, newTR, nvols, newvols",
+    [(2.00000, 6.0, 90, 30),])
```

Comment:
With the precise assertions at the end of this test, I expect this can't actually be parametrized outside some equivalence class. If it can't be parametrized, let's just move these inside. If it can, feel free to ignore this.

Reply:
i could parameterize this potentially, but i will move it inside for now.